docs/sources/write/style-guide/ux-writing/index.md

```diff
@@ -83,7 +83,7 @@ Use active voice to make clear who is performing the action.
 
 ### Use sentence case in UI elements
 
-Capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns.
+Capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns ([product names]({{< relref "../capitalization-punctuation#capitalizing-proper-nouns" >}}) are considered proper nouns).
```

Comment:
```suggestion
Capitalize only the first word in the title, the first word in a subheading after a colon, and any ([proper-noun)]({{< relref "../capitalization-punctuation#capitalizing-proper-nouns" >}}) product names.
```

Reply:
I think this suggestion obscures the intent of clarifying that product names are proper nouns.